ground/src/main/java/com/google/android/ground/ui/home/HomeScreenViewModel.kt

```diff
@@ -34,14 +34,15 @@ class HomeScreenViewModel
 @Inject
 internal constructor(
   private val navigator: Navigator,
-  private val surveyRepository: SurveyRepository,
+  surveyRepository: SurveyRepository,
 ) : AbstractViewModel() {
 
   private val _openDrawerRequests: MutableSharedFlow<Unit> = MutableSharedFlow()
   val openDrawerRequestsFlow: SharedFlow<Unit> = _openDrawerRequests.asSharedFlow()
 
+  // TODO(#1730): Allow tile source configuration from a non-survey accessible source.
   val showOfflineAreaMenuItem: LiveData<Boolean> =
```

Comment:
I think you can just use `= MutableLiveData(true)` here instead of the flow at this point.

Reply:
Thank you! Done.